docs/csharp/misc/cs0245.md

```diff
@@ -10,9 +10,9 @@ ms.assetid: 3f2beb2f-a510-4568-9d11-bb1f65066acd
 ---
 # Compiler Error CS0245
 
-Destructors and object.Finalize cannot be called directly. Consider calling IDisposable.Dispose if available.  
+Finalizers and object.Finalize cannot be called directly. Consider calling IDisposable.Dispose if available.  
```

Comment:
@BillWagner @Uzivatel919 
This change should be reverted, as this is the compiler message itself (the following screenshot is from C# 10):
![image](https://user-images.githubusercontent.com/15279990/133134937-74904cf3-ce4e-4ae7-8fb8-18a64f269c5d.png)